daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -62,16 +63,17 @@ message PackageRef {
 
     // An index into `interned_package_ids` of the Package containing
     // this reference.
-    uint64 interned_id = 3; // *Available in versions >= 1.6*
+    int32 interned_id = 3; // *Available in versions >= 1.6*
   }
 }
 
 // A `name`, e.g. Util.Either.isLeft
 message DottedName {
-
-  // *Must be a non-empty list of a valid identifiers*
+  // *Must be a non-empty list of a valid identifiers unless `segments_interned_id` is set.*
   repeated string segments = 1;
```

Comment:
```suggestion
   repeated string segments = 1; // Deprecated in version >= 1.dev
```
I would like to push for that.
I volunteer to modify the scala encoder in an upcomming PR.

Reply:
postponed to upcoming PR.